src/Rendering/babylon.lineEdgesRenderer.ts

```diff
@@ -96,21 +26,13 @@ module BABYLON {
                 return;
             }
 
-            // First let's find adjacencies

```

Comment:
Can you explain why this code was removed?

Reply:
the question should be : why it has been added first ?
My gut feeling is that this implementation starts with a quick'n dirty copy/paste of the `EdgesRenderer` implementation, with as little change as possible to make it work for LinesMesh.

Unless I'm wrong and I'm missunderstanding what a `LinesMesh` is (only lines, no faces at all), there is no point having and computing something like `FaceAdjacencies` here. We already know what are the lines to be rendered, as the current mesh is made of lines only.

Reply:
Well you are right... Just to be sure did you run some tests? can you share a playground?

Reply:
I test it in the app I'm developping, but I can build a small PG for that. will do either later today or tomorow

Reply:
Thanks so I will merge and add the test as visual validation